docs/guide/getting-started/installation.md

```diff
@@ -10,6 +10,8 @@ Read more about it on the [nix page](./nix#astal)
 
 maintainer: [@kotontrion](https://github.com/kotontrion)
 
+<!--TODO: fix aur package names-->
```

Comment:
@kotontrion How do you plan to name the AUR packages? The Lua package might need an update since it was moved to lang/lua from core/lua. The gjs package was also moved out of the core meson build, so an astal-js package might be beneficial when packaging ags v2 since it will depend on it.

Reply:
the io package will be named `libastal-io` in accordance to the other libs. For the widgets one i'm not sure yet, haven't gave it any thought, but I'm leaning to reusing `libastal` for the gtk3 one and the gtk 4 version can then be named `libastal-gtk4`. (All of them will have the -git suffix, as there is no release yet).

I will update the packages after this was merged. I can fill this in the names in the docs when i have done so.